modules/http/src/test/java/org/mule/module/http/functional/requester/HttpStandardRevocationConfig.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.http.functional.requester;
+
+import org.mule.tck.junit4.FunctionalTestCase;
+
+import org.junit.Test;
+
+public class HttpStandardRevocationConfig extends FunctionalTestCase
+{
+    @Override
+    protected String getConfigFile()
+    {
+        return "http-requester-standard-revocation-config.xml";
+    }
+
+    @Test
+    public void acceptsEmptyTrustStore() throws Exception
+    {
+
```

Comment:
Why is this test empty?

Reply:
I used it to test with google, but it would require accessing an external server. If that's acceptable I'll just assert that it fetches a query.